src/test/spec/unified_runner/operation.rs

```diff
@@ -1674,25 +1672,21 @@ impl TestOperation for RunCursorCommand {
         async move {
             let command = self.command.clone();
             let db = test_runner.get_database(id).await;
-            let cursor_type = match self
-                .cursor_type
-                .clone()
-                .unwrap_or_default()
-                .to_ascii_lowercase()
-                .as_str()
-            {
-                "tailable" => Some(CursorType::Tailable),
-                "tailableawait" => Some(CursorType::TailableAwait),
-                "nontailable" => Some(CursorType::NonTailable),
-                _ => None,
-            };
+
+            let max_time = self.options.as_ref().and_then(|opts| opts.max_time);
+            let cursor_type = self.options.as_ref().and_then(|opts| opts.cursor_type);
+            let selection_criteria = self.options.as_ref().and_then(|opts| opts.selection_criteria.clone());
+            let comment = self.options.as_ref().and_then(|opts| opts.comment.clone());
+            let batch_size = dbg!(self.options.as_ref().and_then(|opts| opts.batch_size));
+
             let options = RunCursorCommandOptions::builder()
```

Comment:
Rather than reconstructing the options here you should be able to pass in `self.options.clone()` directly where this variable is used instead.